x/dogfood/client/cli/query.go

```diff
@@ -60,3 +62,151 @@ func CmdQueryParams() *cobra.Command {
 
 	return cmd
 }
+
+func CmdQueryOptOutsToFinish() *cobra.Command {
```

Comment:
As we discussed before, the `opt_out` might not depend on the maturity of the undelegations, right? So should we remove these functions?

Reply:
I am still confused. We should speak once. But there is certainly some refactoring here that can be done.